RadarSDK/RadarAPIClient.m

```diff
@@ -232,21 +252,13 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
 
                             [[RadarDelegateHolder sharedInstance] didFailWithStatus:status];
 
-                            return completionHandler(status, nil, nil, nil, nil);
+                            return completionHandler(status, nil, nil, nil, nil, nil);
                         }
         
                         [Radar flushLogs];
                         [RadarState setLastFailedStoppedLocation:nil];
 
-                        id metaObj = res[@"meta"];
```

Comment:
like how you moved the serialization into it's own function, much more readable now